sonar/common/jsonschemas/identifiedby-v1.0.0.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
$schema seems missing.

Reply:
Is that mandatory for json schema parts? The property doesn't exist here, for example https://github.com/rero/rero-ils/blob/dev/rero_ils/jsonschemas/common/cantons-v0.0.1.json.